crates/holochain/src/core/sys_validate/error.rs

```diff
@@ -144,17 +144,13 @@ pub enum ValidationOutcome {
     UpdateTypeMismatch(EntryType, EntryType),
     #[error("Update original {0:?} doesn't match the {1:?} in the update")]
     UpdateHashMismatch(EntryHash, EntryHash),
-    #[error("Signature {0:?} failed to verify for Action {1:?}")]
-    VerifySignature(Signature, Action),
+    // #[error("Signature {0:?} failed to verify for Action {1:?}")]
+    // VerifySignature(Signature, Box<Action>),
```

Comment:
Was this intentional? Should it be removed?

Reply:
It should be, it came up with no uses by my IDE so checked by commenting it out but then forgot to actually delete it